pkg/logging-reconciler/logging_reconciler.go

```diff
@@ -40,12 +40,16 @@ func (l *LoggingReconciler) reconcileCreate(ctx context.Context, lc loggedcluste
 
 	if !controllerutil.ContainsFinalizer(lc, key.Finalizer) {
 		logger.Info("adding finalizer", "finalizer", key.Finalizer)
+
 		controllerutil.AddFinalizer(lc, key.Finalizer)
-		err := l.Client.Patch(ctx, lc.GetObject())
+		// We choose to make a Patch instead of an Update to avoid error when the object is updated by another operator.
```

Comment:
Maybe we want to do that in the o11yop as well?

Reply:
Yes this seems to solve the issue, so we should port this to observability operator. I would suggest first we release this and wait to see if it works as expected.

Reply:
Yes... same errors on that operator